endpoints/openrtb2/video_auction.go

```diff
@@ -510,10 +510,14 @@ func mergeData(videoRequest *openrtb_ext.BidRequestVideo, bidRequest *openrtb.Bi
 
 func createBidExtension(videoRequest *openrtb_ext.BidRequestVideo) ([]byte, error) {
 
-	inclBrandCat := openrtb_ext.ExtIncludeBrandCategory{
-		PrimaryAdServer: videoRequest.IncludeBrandCategory.PrimaryAdserver,
-		Publisher:       videoRequest.IncludeBrandCategory.Publisher,
+	var inclBrandCat *openrtb_ext.ExtIncludeBrandCategory
```

Comment:
I assume so, but good to verify that it is safe that `inclBrandCat` can be `nil`.

Reply:
@hhhjort this works with Impbus, we have ``` "includebrandcategory":null ``` in extention now. 
Before we had ``` "includebrandcategory": {"primaryadserver": 0, "publisher": ""}```. 
Please let me know if we need to confirm it and who to ask.